recipes/bigfile/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
```

Comment:
For simple python installs, you can dispense with the `bld.bat` and `build.sh` and use the `script` section in the recipe yaml file:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26